CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ Since v0.0.10, this Changelog is formatted according to the [Common Changelog][c
 
 ### Added
 
+- ([#721](https://github.com/demos-europe/demosplan-ui/pull/721)) Pass the allowEmpty prop to vue-multiselect, which prevents the deselection of values ([@sakutademos](https://github.com/sakutademos))
```

Comment:
this should be under:
### Fixes
since this is a bug fix and not adding a feature

Reply:
from a demosplan-ui perspective, it is a new feature of the DpMultiselect component.

Reply:
makes sense, I agree